pkgs/top-level/all-packages.nix

```diff
@@ -35947,4 +35947,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  rofi-top = callPackage ../applications/misc/rofi-top { };
```

Comment:
Please sort this.

Reply:
Done I put it next to the other rofi plugins